wire-ios-data-model/Tests/Source/Model/Conversation/ZMConversationTests+MLS.swift

```diff
@@ -56,11 +57,56 @@ final class ZMConversationTests_MLS: ZMConversationTestsBase {
         }
     }
 
-    private func createConversation(groupID: MLSGroupID) -> ZMConversation? {
-        let conversation = ZMConversation.insertNewObject(in: syncMOC)
+}
+
+// MARK: - Migration releated fetch requests
+
+final class ZMConversationTests_MLS_Migration: ModelObjectsTests {
+
+    func test_fetchAllTeamGroupProteusConversations() throws {
+        // Given
+        let conversations = try syncMOC.performGroupedAndWait { syncMOC in
```

Comment:
is there a specific reason to use `performGroupedAndWait` instead of `performAndWait` ?

Reply:
No, the grouped-variant was already in place in the other two test cases, see [here](https://github.com/wireapp/wire-ios/pull/666/files/7a3ae0e5ca8cc7812ab0b7702585e38ce5f2934b#diff-103bd52f5d93d2cf625e9cbec1d2914ce26c847a2e44768e8880a928d6b4e67cL30).